docker/zap-baseline.py

```diff
@@ -94,6 +94,7 @@ def usage():
     print('    -s                short output format - dont show PASSes or example URLs')
     print('    -T                max time in minutes to wait for ZAP to start and the passive scan to run')
     print('    -z zap_options    ZAP command line options e.g. -z "-config aaa=bbb -config ccc=ddd"')
+    print('    -I                Do not return failure on warning')
```

Comment:
Minor nits - I'd add this just after the '-i' option, and the other help texts all start with lowercase.